### PR TITLE
 Display repo description issue#41

### PR DESCRIPTION
**Pull request by pratikshaborle**
image

### CODE REVIEW

File: themes/lucy/layouts/partials/repo-box.html [hunk 1]
```diff
@@ -14,6 +14,9 @@
       {{ $count := len .issues }}
       <span class="hidden md:inline text-sm border px-3 py-1 ml-2 rounded-full border-green-500 text-green-500 font-semibold">{{ $count }} issue{{if gt $count 1}}s{{end}}</span>
     </div>
+    <div class="flex-row flex text-sm py-1 font-mono">
+      <span class="font-semibold text-gray-600">Description: </span>{{ .description }}
```
[the remark this answers is not in the document]
Comment (sanketsaurav) [reply themes/lucy/layouts/partials/repo-box.html:18]: Also, there's no need for the `<span>` here. The description can directly be added inside the parent `<div>`.

File: themes/lucy/layouts/partials/repo-box.html [hunk 2]
```diff
@@ -14,6 +14,9 @@
       {{ $count := len .issues }}
       <span class="hidden md:inline text-sm border px-3 py-1 ml-2 rounded-full border-green-500 text-green-500 font-semibold">{{ $count }} issue{{if gt $count 1}}s{{end}}</span>
     </div>
+    <div class="flex-row flex text-sm py-1 font-mono">
```
Comment (sanketsaurav) [inline themes/lucy/layouts/partials/repo-box.html:17]: Do not use monospace font here, since this is normal text.

**Comment by pratikshaborle** [comment]
Hi @sanketsaurav, 
Updated the PR with the changes as you've suggested.

File: themes/lucy/layouts/partials/repo-box.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (pratikshaborle) [reply themes/lucy/layouts/partials/repo-box.html]: > Also, can you please add a better screenshot?

image

**Pull request merged**